### PR TITLE
Fix fetching objectIds arrays using predicate.

### PR DESCRIPTION
**Pull request by kostiakoval**
Sorry my false. Forgot to use predicate. 
Ohh :(

### CODE REVIEW

**Comment by kostiakoval** [comment]
master of RP @NSElvis  :) :cake:

File: Tests/Tests/Tests.m [hunk 1]
```diff
@@ -12,6 +12,18 @@ @interface Tests : XCTestCase
 
 @implementation Tests
 
+- (User *)insertUserrWithRemoteID:(NSNumber *)remoteID
```
Comment (3lvis) [inline Tests/Tests/Tests.m]: There's a typo on `Userr` :)

File: Tests/Tests/Tests.m [hunk 2]
```diff
@@ -12,6 +12,18 @@ @interface Tests : XCTestCase
...
+                        localID:(NSString *)localID
+                           name:(NSString *)name inContext:(NSManagedObjectContext *)context
```
Comment (3lvis) [inline Tests/Tests/Tests.m]: You you mind moving `inContext:(NSManagedObjectContext *)context` to a new line also?

**Comment by kostiakoval** [comment]
@NSElvis  done

**Pull request merged**